community/OM-Operator-Management/cnv-samples/cnv-operator-install/policy-cnv.yaml

```diff
@@ -0,0 +1,179 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
```

Comment:
A `policy.open-cluster-management.io/description` annotation would be nice, but is not necessary.